pkg/virt-handler/node-labeller/node_labeller.go

```diff
@@ -73,6 +73,7 @@ type NodeLabeller struct {
 	cpuModelVendor          string
 	volumePath              string
 	domCapabilitiesFileName string
+	realtimeCapable         bool
```

Comment:
I'd prefer to avoid mixing method and field names. 
Perhaps simply `realtime` in similar fashion to `SEV`? It may not be a complex type (with the `Supported` boolean), but it achieves the same thing.
Maybe when `isNodeRealtimeCapable` becomes a more sophisticated check it would be appropriate to grant it its own type and matching labels, but that extends beyond this PR.

Reply:
Done.